charts/secrets-store-sync-controller/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: secrets-store-sync-controller
 version: 0.0.1
 appVersion: v0.0.1
-kubeVersion: ">=1.27.0"
+kubeVersion: ">=1.27.0-0"
```

Comment:
Could you please remove this? We only make changes to the `manifest_staging` directory. When we cut a release, we move the changes from `manifest_staging` to the `charts` directory.

Reply:
Thanks. Done.